cmd/templ/main.go

```diff
@@ -298,6 +298,8 @@ Args:
     Set log verbosity level. (default "info", options: "debug", "info", "warn", "error")
   -w
     Number of workers to use when formatting code. (default runtime.NumCPUs).
+  -fail
+    Fails with an error exit code if files are changed for use in CI.
```

Comment:
Small nit here, but I think we can be specific with the error code rather than "an error exit code".

```suggestion
    Fails with exit code 1 if files are changed. (e.g. in CI)
```